shared/src/main/scala/org/jetbrains/sbt/structure/data.scala

```diff
@@ -42,26 +42,29 @@ case class StructureData(sbtVersion: String,
  * @param basePackages List of packages to use as base prefixes in chaining
  * @param target Compiler output directory (value of `target` key)
  */
-case class ProjectData(id: String,
-                       buildURI: URI,
-                       name: String,
-                       organization: String,
-                       version: String,
-                       base: File,
-                       packagePrefix: Option[String],
-                       basePackages: Seq[String],
-                       target: File,
-                       configurations: Seq[ConfigurationData],
-                       java: Option[JavaData],
-                       scala: Option[ScalaData],
-                       compileOrder: String,
-                       dependencies: DependencyData,
-                       resolvers: Set[ResolverData],
-                       play2: Option[Play2Data],
-                       settings: Seq[SettingData],
-                       tasks: Seq[TaskData],
-                       commands: Seq[CommandData]
-                      )
+case class ProjectData(
```

Comment:
Please describe two new parameters in ScalaDoc
```
testSourceDirectories: Seq[File],
mainSourceDirectories: Seq[File
  ```

Reply:
Are these parameters needed anyway?
Isn't data in `ConfigurationData` enough?
If not, then don't we need `main/testResourceDirectories` as well?

Reply:
> Isn't data in ConfigurationData enough?

Unfortunately not :< Sources  are kept in `ConfigurationData` in the form of full paths to the directories e.g. `/Users/azdrojowa/IdeaProjects/sharedSources/src/main/scala-2.13`. In such case, after getting long list of directories it will be needed to create some heuristics to know what should be the root of all these paths. 
In simple sbt projects it is trivial, but with custom source directories it is no that simple. 
Without prod/test sources separation we didn't care too much what was the root path for `ContentRootNode` because it has always been the base path of a project. Now, the situation is different. 

> If not, then don't we need main/testResourceDirectories as well?
 
 I think that no. I know that at first glance, keys `sourceDirectory` + `sourceDirectories` and `resourceDirectory` + `resourceDirectories` seem analogous but it looks different in practice. 
 Let's take a look at the screenshot. When I call `resourceDirectories` I am getting (among others) what is returned from `resourceDirectory`. Key `resourceDirectories` (more precisely `managedResourceDirectories` and `unmanagedResourceDirectories`) is called when creating `ConfigurationData` so it is not needed  to call `resourceDirectory` separately at some other place. 
I did an additional test in which I added in `build.sbt` some custom `unmanagedResourceDirectories` and override `resourceDirectory` key. As a result key `unmanagedResourceDirectories` returned what was saved in `resourceDirectory` and `unmanagedResourceDirectories`. 

An analogous situation does not apply in the case of sources. 
 
<img width="901" alt="Screenshot 2024-05-21 at 17 44 43" src="https://github.com/JetBrains/sbt-structure/assets/33667003/570a6328-491c-4494-b307-e6a3ab4745e7">

Reply:
> Unfortunately not :< Sources are kept in ConfigurationData in the form of full paths to the directories e.g. .....

Ok, let's add a _short_ sentence in Scaladoc mentioning the difference in the content of these fields. An example would be nice to have. Doing it just for "mainSourceDirectories" would be enough, to avoid too much duplicateopn